### DISCUSSION TITLE
Unable to view the e-books

### DISCUSSION THREAD

**Discussion by Coder-Vasen** [johnfactotum/foliate#1324] [Q&A]
I was amazed with foliate's UI design. I wanted to try it for the first time. I installed it on **snap** .
Version _3.1.1_ was installed. When I initially opened I could able to see some broken icon images.
![image](https://github.com/johnfactotum/foliate/assets/69237691/4cbdeb49-ce87-45ab-ba01-26b7b7866f44)

I tried opening the **EPUB** file, but I couldn't able to views its pages.
![image](https://github.com/johnfactotum/foliate/assets/69237691/3b38c347-d9b5-4504-936f-0627edce3496)
![image](https://github.com/johnfactotum/foliate/assets/69237691/9bf57fa8-419e-43fc-a3c0-b20932843ad5)

I am using **Zorin OS** with gnome version of 3.38. I read the project depend on gtk4. I'm not sure will it work as intended on gnome-3 as I am not familiar with Linux.

**Comment by johnfactotum** [comment]
See https://github.com/johnfactotum/foliate/blob/gtk4/docs/troubleshooting.md
